pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -116,19 +116,6 @@ func TestTryGetPod(t *testing.T) {
 
 func TestMakePod(t *testing.T) {
 	names.TestingSeed()
-	subPath := "subpath"
-	implicitVolumeMountsWithSubPath := []corev1.VolumeMount{}
-	for _, vm := range implicitVolumeMounts {
-		if vm.Name == "workspace" {
-			implicitVolumeMountsWithSubPath = append(implicitVolumeMountsWithSubPath, corev1.VolumeMount{
-				Name:      vm.Name,
-				MountPath: vm.MountPath,
-				SubPath:   subPath,
-			})
-		} else {
-			implicitVolumeMountsWithSubPath = append(implicitVolumeMountsWithSubPath, vm)
-		}
-	}
```

Comment:
do you have any idea what this was trying to test and why it's not needed now?

Reply:
It was deleted in the large resource refactor:
https://github.com/tektoncd/pipeline/commit/7a128156f873c82cb6116f95328a6b11833f6a4c